docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -34,14 +34,32 @@ If you see the following warnings while running `kubeadm init`
 
 Then you may be missing ebtables and ethtool on your Linux machine. You can install them with the following commands: 
 
+- For ubuntu/debian users, try `apt install ebtables ethtool`.
+- For CentOS/Fedora users, try `yum install ebtables ethtool`.
+
+#### kubeadm blocks waiting for `control plane` during installation
+
+If you notice that `kubeadm init` hangs after printing out the following line
+
 ```
-# For ubuntu/debian users, try 
-apt install ebtables ethtool
+[apiclient] Created API client, waiting for the control plane to become ready                          
+```
+
+You may want to first check if your node has network connection problem.
+Another reason that `kubeadm init` hangs could be that the default cgroup driver configuration
+for the kubelet differs from that used by Docker.
 
-# For CentOS/Fedora users, try 
-yum install ebtables ethtool
+Check the system log file (e.g. `var/log/message`) or examine the output from `journalctl -u kubelet`.
+If you see something like the following
+
+```
+error: failed to run Kubelet: failed to create kubelet: 
+misconfiguration: kubelet cgroup driver: "systemd" is different from docker cgroup driver: "cgroupfs"
```

Comment:
can we embed this in the above "docker cgroup" bullet point instead of splitting the point over numerous paragraphs